tailwind.config.js

```diff
@@ -64,8 +69,6 @@ module.exports = {
     require(dir + '/utilities/accessibility/accessibily-hidden.js')(),
     require(dir + '/utilities/accessibility/sr-only-element.js')(),
     require(dir + '/utilities/accessibility/sr-only-text.js')(),
-    require(dir + '/utilities/flex/flex-container.js')(),
-    require(dir + '/utilities/flex/flex-container-row.js')(),
```

Comment:
TW has changed their flex grid, these that we added no longer work as intended. Let's go with the TW convention - for most parts, their CSS grid is better for what we need and if we aren't support IE then there is no issue using CSS grid to replace our flex classes.

Reply:
We can always add to this where we need to.